lib/vet360/models/address.rb

```diff
@@ -140,7 +140,9 @@ def in_json
             zipCode5: @zip_code_suffix,
             originatingSourceSystem: SOURCE_SYSTEM,
             sourceDate: @source_date,
-            vet360Id: @vet360_id
+            vet360Id: @vet360_id,
+            effectiveStartDate: @effective_start_date,
+            effectiveEndDate: @effective_end_date
```

Comment:
Should we remove these fields from serialization if they are nil? I'm wondering if since we aren't requiring the frontend to always pass both values, we may be clearing out existing values. However I am trying to think of a situation in where this would occur, because we always default start date, and would at this point in time not setting an effective end date in the future. However that wouldn't prevent external systems from setting values that may get nulled out by our PUT. What do you guys think @emersonthis @hpjaj?

Reply:
Hmm good question. The other side of this is that if we remove them when empty there won't be a way to remove an effectiveEndDate. They'd have to "delete" it and start over I guess. But that's probably something we can worry about later.

Reply:
After further discussion / research we've determined the current implementation is correct.

@rlbaker Let me know if you have any other concerns || approval